e2e_testing/torchscript/basic.py

```diff
@@ -925,3 +925,54 @@ def forward(self, input):
 @register_test_case(module_factory=lambda: NumelZeroRankModule())
 def NumelZeroRankModule_basic(module, tu: TestUtils):
     module.forward(torch.randint(10,[]))
+
+class BoolReturnFalseModule(torch.nn.Module):
```

Comment:
You might want to call these "BoolTensor" instead of just "Bool", since there is also returning of `!torch.bool` (python scalar booleans) eventually.